cmd/libvirt-actuator/main.go

```diff
@@ -309,6 +309,12 @@ func bootstrapCommand() *cobra.Command {
 				glog.Info(msg)
 			}
 
+			clusterFramework.MachineControllerImage = "openshift/origin-libvirt-machine-controllers:v4.0.0"
+			clusterFramework.MachineManagerImage = "openshift/origin-libvirt-machine-controllers:v4.0.0"
+			// TODO(jchaloup): once docker.io/openshift/origin-machine-api-operator gets updated, switch back to it.
```

Comment:
It should get an update today, as https://github.com/openshift/machine-api-operator/pull/159 is on its way.

Reply:
Who is responsible for making the updates?

Reply:
prow

Reply:
How often is it done?

Reply:
As far as I remember it should be done every couple of hours

Reply:
The latest mao docker images is two weeks old :-|

Reply:
It seems like like CI image mirroring responsible for uploading images to docker hub was down between ~13 days ago and ~43 minutes ago.